deps/fast_float/Makefile

```diff
@@ -0,0 +1,25 @@
+# Fallback to gcc/g++ when $CC or $CXX is not in $PATH.
+CC ?= gcc
+CXX ?= g++
+
+CFLAGS=-Wall -O3
+# This avoids loosing the fastfloat specific compile flags when we override the CFLAGS via the main project
+FASTFLOAT_CFLAGS=-std=c++11 -DFASTFLOAT_ALLOWS_LEADING_PLUS
```

Comment:
We are already using c++20
> set(CMAKE_CXX_STANDARD 20)

Wouldn't it be better if we align on the same standard if possible?

Reply:
Makefile was removed, we are using cmake